### PR TITLE
EXTERMINATE EXTERMINATE EXTERMINATE

### PR DESCRIPTION
**Pull request by optimumtact**

### CODE REVIEW

**Comment by Thunder12345** [comment]
Close this shit, save /tg/ culture

**Comment by optimumtact** [comment]
if this is tg culture then we died a long time ago

File: code/game/objects/structures/fireaxe.dm [hunk 1]
```diff
@@ -130,7 +130,7 @@
 		locked = !locked
 		update_icon()
 
-/obj/structure/fireaxecabinet/verb/toggle_open() //nice name, huh? HUH?! -Erro //YEAH -Agouri
```
Comment (Jalleo) [inline code/game/objects/structures/fireaxe.dm]: Damn humans being exterminated by the oranges we need a doctor to stop this :laughing:

**Comment by Thunder12345** [comment]
We did, all we can do now is protect what we have left

File: code/game/objects/structures/fireaxe.dm (hunk 1 above)
Comment (Dorsisdwarf) [inline code/game/objects/structures/fireaxe.dm]: :scream_cat:

**Pull request merged**